test/studies/ml/performance/mnist_cnn_batch.perfkeys

```diff
@@ -0,0 +1 @@
+time:
```

Comment:
What reports the time here? The library? What flag is used to enable time measurement?

Reply:
in `ml/lib/MNISTTester.chpl`, `config param perfTest = false;` will enable the time reporting.

Reply:
Could you maybe put a comment explaining that in this test's .chpl file?